firestore/firestore_snippets/save.go

```diff
@@ -22,10 +23,11 @@ func addDocAsMap(ctx context.Context, client *firestore.Client) error {
 		"country": "USA",
 	})
 	if err != nil {
-		return err
+		// Handle the error correctly here.
```

Comment:
Say `TODO: Handle the error correctly.`?

Reply:
It's not really a TODO, it's a comment for the user instructing them that they should handle the error inside that block.